frontend/.eslintrc.js

```diff
@@ -23,4 +23,12 @@ module.exports = {
   ],
   rules: {
   },
+  overrides: [{
+    files: ["**/*.test.js", "**/*.test.jsx"],
+    env: {
+      browser: true,
+      es6: true,
```

Comment:
```suggestion
```

Reply:
these are already in line 3-4, those are global, these will override those global rules, so only jest is needed here